terraform/azure/variables.tf

```diff
@@ -881,3 +881,22 @@ variable "iscsi_remote_python" {
   default     = "/usr/bin/python3"
 }
 
+# Network peering variables
+variable "enable_az_vnet_peering" {
```

Comment:
Is it strictly needed? Cannot we use `len(ibsm_rg) != 0` in place of it? Do we need a variable to expose to the external and not just infere indirectly if user like to create peering from presence of absence of other variables?